wp-dependency-installer.php

```diff
@@ -264,13 +293,13 @@ public function admin_init() {
 				} elseif ( $this->is_installed( $slug ) ) {
 					if ( $is_required ) {
 						$this->notices[] = $this->activate( $slug );
-					} else {
+					} elseif ( $suggest_active ) {
 						$this->notices[] = $this->activate_notice( $slug );
 					}
 				} else {
 					if ( $is_required ) {
 						$this->notices[] = $this->install( $slug );
-					} else {
+					} elseif ( $suggest_install ) {
 						$this->notices[] = $this->install_notice( $slug );
```

Comment:
That is: show an `install now` notice only if the plugin is `optional` and the developer only cares that the plugin would **maybe installed** on the system (and not just `always installed`)

Reply:
I think we're getting into semantics. If the dev is only suggesting a plugin be installed that's what the timeout setting is for. Set a longer timeout.